cmd/bb_runner/install

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -eu
+
+echo "$*"
+
+# Can pass the destination directory as the first argument. By default
+# it will use /bb
+dest="${1:-/bb}"
+
+cp /bb_runner /tini "$dest"
```

Comment:
Maybe we should place all these files in a subdirectory, so they can be copied into into /bb without requiring constant changes to this script?

Reply:
I thought about this, and then I thought "how often do we really add new binaries to bb_runner?"

Notably it would require changing the bb_runner and tini layers, so I'd have to duplicate those to avoid changing how the ubuntu image works, which seemed like more trouble than it's worth for a thing that might never happen.